toptal/gitignore.io
# [TRA-1617] Update gitignore template submodule

# Description
This PR synchronizes templates from https://github.com/toptal/gitignore

https://github.com/toptal/gitignore/compare/66f88bd...1a8b5b5

# How to test

- `docker-compose up --build`
- go to http://localhost:8080/api/zig
- verify `zig-out/` included in the response
